packages/mds-jurisdiction/tests/api.spec.ts

```diff
@@ -0,0 +1,165 @@
+import { SCOPED_AUTH } from '@mds-core/mds-test-data'
+import { Jurisdiction } from '@mds-core/mds-types'
+import { api } from '../api'
+import { JURISDICTION_API_DEFAULT_VERSION } from '../types'
+
+const request = supertest(ApiServer(api))
+
+const [JURISDICTION0, JURISDICTION1, JURISDICTION2] = [uuid(), uuid(), uuid()].map(jurisdiction_id => ({
+  jurisdiction_id,
+  agency_key: `agency-key-${jurisdiction_id}`,
+  agency_name: `Agency Name ${jurisdiction_id}`,
+  geography_id: uuid()
+}))
+
+describe('', () => {
+  before(async () => JurisdictionService.initialize())
+
+  it('Create Single Jurisdiction', async () => {
+    const result = await request
+      .post('/jurisdictions')
+      .set('Authorization', SCOPED_AUTH(['jurisdictions:write']))
+      .send(JURISDICTION0)
+      .expect(201)
+    test.object(result.body).hasProperty('version', JURISDICTION_API_DEFAULT_VERSION)
+    test.object(result.body.jurisdiction).hasProperty('timestamp')
+    test.object(result.body.jurisdiction).hasProperty('jurisdiction_id', JURISDICTION0.jurisdiction_id)
+  })
+
+  it('Create Single Jurisdiction (forbidden)', async () => {
+    await request
+      .post('/jurisdictions')
+      .send(JURISDICTION0)
+      .expect(403)
+  })
+
+  it('Create Single Jurisdiction (conflict)', async () => {
+    await request
+      .post('/jurisdictions')
+      .set('Authorization', SCOPED_AUTH(['jurisdictions:write']))
+      .send(JURISDICTION0)
+      .expect(409)
+  })
+
+  it('Create Single Jurisdiction (validation error)', async () => {
+    const { agency_key, ...dto } = JURISDICTION1
+    await request
+      .post('/jurisdictions')
+      .set('Authorization', SCOPED_AUTH(['jurisdictions:write']))
+      .send(dto)
+      .expect(400)
+  })
+
+  it('Create Multiple Jurisdictions', async () => {
+    const result = await request
+      .post('/jurisdictions')
+      .set('Authorization', SCOPED_AUTH(['jurisdictions:write']))
+      .send([JURISDICTION1, JURISDICTION2])
+      .expect(201)
+    test.object(result.body).hasProperty('version', JURISDICTION_API_DEFAULT_VERSION)
+    test.object(result.body.jurisdictions).hasProperty('length', 2)
+    test
+      .value(result.body.jurisdictions.map((jurisdiction: Jurisdiction) => jurisdiction.jurisdiction_id))
+      .is([JURISDICTION1.jurisdiction_id, JURISDICTION2.jurisdiction_id])
+  })
+
+  it('Get One Jurisdiction', async () => {
+    const result = await request
+      .get(`/jurisdictions/${JURISDICTION2.jurisdiction_id}`)
+      .set('Authorization', SCOPED_AUTH(['jurisdictions:read']))
+      .expect(200)
+    test.object(result.body).hasProperty('version', JURISDICTION_API_DEFAULT_VERSION)
+    test.object(result.body.jurisdiction).hasProperty('jurisdiction_id', JURISDICTION2.jurisdiction_id)
+  })
+
+  it('Get One Jurisdiction (not found)', async () => {
+    await request
+      .get(`/jurisdictions/${uuid()}`)
+      .set('Authorization', SCOPED_AUTH(['jurisdictions:read']))
+      .expect(404)
```

Comment:
How was this passing? I didn't see that handler returning a 404.

Reply:
It does. `api.ts` line `103`

Reply:
More confusion on my end, thanks